### PR TITLE
Use optional dockerfile

### LINKED ISSUE #305: ghcr.io are not found for "special" dockers
For Android e.g translations-plain is needed.
Which is configured on hub.docker.com like:

Type: Tag
Source: /translations-plain-.*/
Dockerfile location: /translations/Dockerfile-Plain

Can we have such a workflow? @LukasReschke

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #305 

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
Check https://transifex-sync.nextcloud.com and trigger all missing images after merge

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@ on:
         description: "Path to the folder"
         required: true
         default: 'webdav-apache-php7.3'
+      dockerFile:
+        description: "Optional relative (from folder) path to Dockerfile. Default is $folder/Dockerfile"
+        required: false
+        default: 'Dockerfile'
+      suffix:
+        description: "Option suffix to image name"
```
Comment (skjnldsv) [inline .github/workflows/docker.yml:15]: Have an example of what it looks like?

**Comment by tobiasKaminsky** [comment]
image

> Unable to find image 'ghcr.io/nextcloud/continous-integration-translations-externalandroid-branched:latest

Dockerfile is located in this repo in /translations and named Dockerfile-ExternalAndroid-Branched

**Comment by skjnldsv** [comment]
In the description

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**